src/main/java/org/jabref/gui/BasePanel.java

```diff
@@ -908,10 +908,6 @@ public void updateEntryEditorIfShowing() {
 
     public void markBaseChanged() {
         baseChanged = true;
-        markBasedChangedInternal();
-    }
-
-    private void markBasedChangedInternal() {
```

Comment:
I remember hard discussions when introducing this. That we really need it etc. What a pity that is was not documented. In retrospective, that would have been interesting.

Nice to see that this special function is gone now.